src/benchmarklib/tpch/tpch_queries.cpp

```diff
@@ -894,6 +894,8 @@ const char* const tpch_query_22 =
 
 namespace opossum {
 
+// clang-tidy does not like global non-trivial objects that are not initialized with constexpr
+// NOLINTNEXTLINE(fuchsia-statically-constructed-objects)
```

Comment:
Disable these in clang-tidy?

Reply:
fuchsia is developed so the linux kernel doesn't get in the way of ad-delivery into you phone conversations or something, no?

Disable, I'd say - static objects are cool for us.